dydx-testnet-4/gentx/gentx-Blockdaemon.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Blockdaemon","identity":"","website":"https://blockdaemon.com","security_contact":"","details":""},"commission":{"rate":"0.050000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx10e6g0qc7hmcalqlywm22m0ep33nvjpm6tznemu","validator_address":"dydxvaloper10e6g0qc7hmcalqlywm22m0ep33nvjpm6wu90sd","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"5c2AfqnBUjpw7wjAdejKLIkDB02rRMVyeaQ7iAI8GRg="},"value":{"denom":"adv4tnt","amount":"1000000000000000000000000"}}],"memo":"62230535360e25317617e48d4fdcae911d02565c@172.18.0.5:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A0aUH9Lddt4RVwDLzLCBYxk7JzZLr+cb4cwNnRbzlXPq"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["iLkGCSP0NCnaanH3Dx0OHOIpXHfMlDGr7BucCuPPbkIscfeoBALo0aK6zdawJFiIHjb85MSiOlfcKO7TzHmRYQ=="]}
```

Comment:
@JackPMallon As per the updated https://v4-teacher.vercel.app/validators/preparing_for_genesis the new amount is `500000000000000000000000adv4tnt`